sdk/agrifood/ci.yml

```diff
@@ -8,6 +8,7 @@ trigger:
   paths:
     include:
     - sdk/agrifood/
+    - sdk/resourcemanager/
```

Comment:
please dont create a new ci.mgmt.yml file, refer this https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/advisor/ci.mgmt.yml